src/db.js

```diff
@@ -1,8 +1,8 @@
-async function execQuery(pool, q, args = null, responseFunc = null) {
+async function execQuery(pool, q, args = null, responseFunc = null, db = 'headsup') {
```

Comment:
Why isn't `db` just set by environment variable like every other DB-related variable?

Reply:
Because we request from the headsup database and ambulance website database depending on the query so we can't just have an environment variable because it changes depending on your query.

Reply:
You called?

Reply:
Fair point; however, the DB names should not be hard-coded as strings. If there are two DBs in use, let's define them both as variables.